Developer_Documentation/1.6/code_docs/admin_lead_pages/introduction.md

```diff
@@ -1,25 +1,23 @@
 Introduction
 ============
 
-The Zen Cart Administration Interface contains a number of pages that are used to manage a simple list of items. Examples are the admin->Locations/Taxes/Countries page, the admin->Locations/Taxes/Zones page and many others.
+The Zen Cart Administration Interface contains a number of pages that are used to manage simple lists of items. Examples are the Countries and Zones pages under Locations/Taxes.
 
-What is common amongst these pages is that they initially show a paginated list of entries, and then allow you to add new entries/edit an existing entry and delete entries.
+What is common amongst these pages is that they initially show a paginated list of entries, and then allow you to edit an existing entry, add new entries, and delete entries.
 
 The current pages that manage these lists have a number of problems. They are difficult to create easily, they are almost impossible to customize using add-ons and the code is usually quite messy, with php code interspersed with output HTML.
```

Comment:
Perhaps now is the time to also change this to say `In older versions the admin pages that managed ...` instead of `The current pages that manage...`